Dockerfile

```diff
@@ -1,26 +1,127 @@
-FROM ubuntu:18.04
-
-# install updates
-RUN apt-get update -y && \
-apt-get upgrade -y && \
-apt-get install -y python3.7 python3-pip && \
-cd /usr/bin && ls -lrth python* && \
-unlink python3 && ln -s /usr/bin/python3.7 python3 && \
-python3 --version
-RUN pip3 install --upgrade pip && \
-apt-get install -y git && \
-apt-get install -y wget && \
-git clone https://github.com/diplomacy/research.git && \
-git clone https://github.com/ALLAN-DIP/baseline_bots.git && \
-cd baseline_bots && \
-git checkout -b test_CI remotes/origin/test_CI && \
-pip3 install -r requirements.txt && \
-cd ..
-RUN cd research && \ 
-pip3 install -r requirements.txt && \
-pip3 install -r requirements_dev.txt
-RUN cd ../baseline_bots && \
-pip3 install -e . && \
-python3 -m pytest tests/utils_test.py && \
-python3 -m pytest tests/bot_tests/bots_test.py && \
-python3 -m pytest tests/randomize_order_test.py
+FROM pcpaquette/tensorflow-serving:20190226
+
+WORKDIR /model/src/model_server
+
+RUN apt-get -y update
+RUN apt-get -y install git
+RUN apt-get -y install vim
+RUN apt-get -y install curl
+RUN apt-get -y install htop
+RUN apt-get -y install lsof
+
+# Copy SL model
+RUN mkdir /model/src/model_server/bot_neurips2019-sl_model
+COPY bot_neurips2019-sl_model /model/src/model_server/bot_neurips2019-sl_model 
+COPY run_model_server.sh /model/src/model_server/run_model_server.sh
+RUN chmod 777 /model/src/model_server/run_model_server.sh
+RUN chmod -R 777 /model/src/model_server/bot_neurips2019-sl_model
+
+# TODO: Get this to work for RL model as well
+
+# Clone repos
+RUN git clone https://github.com/SHADE-AI/diplomacy.git
+RUN git clone https://github.com/SHADE-AI/research.git
+RUN git clone --single-branch --branch containerizing https://github.com/ALLAN-DIP/baseline_bots.git
```

Comment:
What exactly is this line doing?

Reply:
It seems like it is only cloning from this containerizing branch, which is not what we want in a CI pipeline.